helpers/apps

```diff
@@ -111,3 +111,68 @@ ynh_remove_apps() {
 		done
 	fi
 }
+
+# Load an app environment in the current Bash shell
+#
+# usage: ynh_install_apps --app="app"
+#     | arg: -a, --app=     - the app ID
+#
+# Requires YunoHost version 11.0.* or higher.
+ynh_load_app_environment() {
```

Comment:
The naming suggests that we only load the app environment, but we rather log as the app user (using the `su` command).

What about: `ynh_spawn_app_shell`? `ynh_start_app_shell`?

(Also thanks so muuuch for the PR, I really look forward using the command :heart:)